addons/api/fnc_setRevealToAI.sqf

```diff
@@ -19,6 +20,8 @@ INFO_2("%1 called with: %2",QFUNC(setRevealToAI),_this);
 
 params ["_var"];
 
+if (!hasInterface) exitWith {false};
```

Comment:
So far it's mostly been doing the same thing just silently exit if run on server/headless client.